ui/app/AppLayouts/Chat/popups/community/CreateCategoryPopup.qml

```diff
@@ -71,84 +71,102 @@ StatusModal {
             bottomPadding: 8
         }
 
-        StatusScrollView {
-            id: scrollView
-
+        Item {
             width: root.width
             height: Math.min(content.height, 300)
-            anchors.horizontalCenter: parent.horizontalCenter
 
-            property ScrollBar vScrollBar: ScrollBar.vertical
+            Item {
 
-            contentHeight: content.height
-            ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
+                anchors.fill: parent
+                anchors.topMargin: -8
+                anchors.bottomMargin: -8
 
-            function scrollBackUp() {
-                vScrollBar.setPosition(0)
-            }
+                clip: true
 
-            Item {
-                id: content
-                width: parent.width
-                height: channelsLabel.height + communityChannelList.height
-
-                Item {
-                    id: channelsLabel
-                    anchors.horizontalCenter: parent.horizontalCenter
-                    width: parent.width - 32
-                    height: 34
-                    StatusBaseText {
-                        text: qsTr("Channels")
-                        anchors.bottom: parent.bottom
-                        anchors.bottomMargin: 4
-                        font.pixelSize: 15
-                        color: Theme.palette.baseColor1
+                StatusScrollView {
+                    id: scrollView
+
+                    anchors.topMargin: parent.anchors.topMargin
+                    anchors.bottomMargin: parent.anchors.bottomMargin
+                    anchors.fill: parent
+
+                    property ScrollBar vScrollBar: ScrollBar.vertical
+                    clip: false
+
+                    contentWidth: scrollView.width - scrollView.leftPadding - scrollView.rightPadding
```

Comment:
this should be calculated in StatusScrollView as availableWidth

Reply:
Fixed, thanks :)